src/test/javascript/spec/component/programming-exercise/programming-exercise-editable-instruction.component.spec.ts

```diff
@@ -51,10 +53,18 @@ describe('ProgrammingExerciseEditableInstructionComponent', () => {
         { testName: 'test3', active: false },
     ];
 
-    beforeEach(async () => {
+    beforeEach(() => {
         return TestBed.configureTestingModule({
-            imports: [TranslateModule.forRoot(), ArtemisTestModule, NgbModule, ArtemisProgrammingExerciseInstructionsEditorModule],
-            declarations: [MockComponent(ProgrammingExerciseInstructionAnalysisComponent), MockComponent(MarkdownEditorComponent)],
+            imports: [TranslateModule.forRoot(), ArtemisTestModule],
```

Comment:
`TranslateModule.forRoot()` can also be replaced with a mock provider.

Reply:
I removed it :+1: